src/content/glossary/index.md

```diff
@@ -123,6 +123,14 @@ A group of at least 128 [validators](#validator) assigned to beacon and shard bl
 
 When numerous nodes (usually most nodes on the network) all have the same blocks in their locally validated best blockchain. Not to be confused with [consensus rules](#consensus-rules).
 
+### consensus client {consensus-client}
```

Comment:
Think you accidentally added your other PR into this one :-)

Reply:
ffs, ok sorry, will try to fix in next commit